### PR TITLE
Create docker scripts for production

### PR DESCRIPTION
**Pull request by GrantPerkins**
Creates a build and start script for ubuntu. Users would only run start.

### CODE REVIEW

File: docker_scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name=axon
+
+name_with_tag="gcperkins/${name}:latest"
```
Comment (AustinShalit) [inline docker_scripts/build.sh]:
```suggestion
name_with_tag="wpilib/${name}:latest"
```

File: docker_scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+docker build  -t ${name} ..
+docker tag ${name} ${name_with_tag}
+docker push ${name_with_tag}
```
Comment (AustinShalit) [inline docker_scripts/build.sh]:
```suggestion
docker push ${name_with_tag}

```

File: docker_scripts/start.sh [hunk 3]
```diff
@@ -1 +1,3 @@
-docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -p 3000:3000 -p 4000:4000 gcperkins/axon:latest
\ No newline at end of file
+xdg-open http://localhost:3000
```
Comment (AustinShalit) [inline docker_scripts/start.sh]: Does this work with WSL?

Comment (GrantPerkins) [reply docker_scripts/start.sh]: Unsure. I can go try. Working on ubuntu 20.04 right now

Comment (GrantPerkins) [reply docker_scripts/start.sh]: I will be making another start script for windows

**Pull request closed without merging**